### PR TITLE
MCKIN-29542 exception handling

### PR DESCRIPTION
**Pull request by mudassir-hafeez**
Handling error with returning None if a user is not enrolled in the course.

### CODE REVIEW

**Comment by mudassir-hafeez** [comment]
@xitij2000 can you please review? I have created a separate PR as I don't have access to push changes into the existing PR in absence of Ahamd. Thanks!
++ @naeem91

**Comment by mudassir-hafeez** [comment]
@viadanna can you please review in absence of Kshitij?
To get more context, check this PR #41

File: setup.py [hunk 1]
```diff
@@ -22,15 +22,15 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-eoc-journal',
-    version='0.9.4',
+    version='0.9.5',
```
Comment (viadanna) [inline setup.py:25]: Can you bump this to '0.10.0' instead as we are dropping Python2 support?

**Review by viadanna — COMMENTED** [review]
@mudassir-hafeez Just a small change to the version and this is good for merging 👍

File: setup.py (hunk 1 above)
Comment (mudassir-hafeez) [reply setup.py]: Addressd!

**Comment by viadanna** [comment]
Good to go 👍

- [x] I tested this on my devstack.
- [x] I read through the code.

**Comment by mudassir-hafeez** [comment]
Thanks @viadanna! can you please create a release tag against `0.10.0`?

**Comment by viadanna** [comment]
Sure @mudassir-hafeez done 👍

**Pull request merged**